.github/CODEOWNERS

```diff
@@ -341,11 +340,10 @@ yarn.lock                                                @getsentry/owners-js-de
 /src/sentry/tasks/symbolication.py                                   @getsentry/owners-native
 /src/sentry/utils/appleconnect/                                      @getsentry/owners-native
 /tests/sentry/tasks/test_low_priority_symbolication.py               @getsentry/owners-native
-
-# Shared ownership of reprocessing as transitionary phase
-/src/sentry/tasks/reprocessing.py                                    @getsentry/owners-ingest @getsentry/owners-native
-/src/sentry/tasks/reprocessing2.py                                   @getsentry/owners-ingest @getsentry/owners-native
-/src/sentry/reprocessing2.py                                         @getsentry/owners-ingest @getsentry/owners-native
+/src/sentry/tasks/reprocessing.py                                    @getsentry/owners-processing
+/src/sentry/tasks/reprocessing2.py                                   @getsentry/owners-processing
+/src/sentry/reprocessing2.py                                         @getsentry/owners-processing
+/src/sentry/tasks/store.py                                           @getsentry/owners-processing
```

Comment:
I don't know why github marks these rules as invalid. Am I missing a typo here? https://github.com/orgs/getsentry/teams/owners-processing

Reply:
I think that team has no write access updated for this repo, which is required for codeowners to work.

Reply:
I think so too. I see it's a top level team in `getsentry`, and I guess it should be under `Staff/Engineering/...`

Reply:
Approval still pending. @mitsuhiko Is this something you can do? For both codeowners and team teams. 

<img width="686" alt="owners-processing_Team_·_Edit_·_General" src="https://user-images.githubusercontent.com/374965/230628098-36947e4a-ce8c-4995-8514-076f0634f2c4.png">

Reply:
@ashwoods thank you, this looks green now. I won't merge the PR without approval from the processing team itself though. In case it looks good to you, could you please give it a stamp?